<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fender PLUG</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="Amplifier">
      <property name="text">
       <string>Amplifier</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="EffectButton1">
        <property name="text">
         <string>Effect 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="EffectButton2">
        <property name="text">
         <string>Effect 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="EffectButton3">
        <property name="text">
         <string>Effect 3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="EffectButton4">
        <property name="text">
         <string>Effect 4</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
